frontend/src/components/About.tsx

```diff
@@ -1,40 +1,66 @@
-import { Button, Container, Divider, Paper, Typography } from '@mui/material'
+import { Box, Button, Container, Divider, Paper, Typography } from '@mui/material'
 import React from 'react'
 import { StyledButton } from './styles'
+import { useLocalStorage } from 'src/hooks/useLocalStorage';
```

Comment:
Why not relative path?

Reply:
Updated! not sure why vscode chose to do absolute for the import